src/algorithms/solvers.jl

```diff
@@ -142,8 +142,8 @@ function _core_msolve(
 
     jl_vnames = Base.unsafe_wrap(Array, res_vnames[], jl_rp_nr_vars)
     vsymbols  = [Symbol(unsafe_string(jl_vnames[i])) for i in 1:jl_rp_nr_vars]
-    #= get possible variable permutation =#
-    perm      = sortperm(vsymbols)
+    #= get possible variable permutation, ignoring additional variables=#
+    perm      = sortperm(vsymbols[1:1:nr_vars])
```

Comment:
Why do you explicitly state the step length here?

```suggestion
    perm      = sortperm(vsymbols[1:nr_vars])
```

Reply:
Sorry, I'm just starting learning Julia and I (stupidly) though that specifying the step length was mandatory..
Thanks!